### PR TITLE
Create "asr-error-robustness.yaml" to release a dataset

### PR DESCRIPTION
**Pull request by anjiefang**
Create "asr-error-robustness.yaml" to release a dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 9de684bf1781789cd448ca9c4a7119572539e809
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c462d51a-cc9d-4611-9b27-f7736961f62c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 597f8d4243e2f1ca15ee49cfe90c94fc23ec7cfc
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=87ec465d-cf0c-4a74-979d-6906fb7e7104%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 96d570d91ac6e8458184246c80fd244fd4df3fe3
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=4153ed9b-8bed-497e-8644-234f42def9e2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by anjiefang** [comment]
Get the following report error:
```
pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:
 - Value 'https://assets.amazon.science/64/94/639ae0c44890837b0f1fbf11ef77/using-phoneme-representations-to-build-predictive-models-robust-to-asr-errors.pdf' does not match pattern '^(https?:\/\/)?([\da-z\.\-\_]+)\.
```
`https://assets.amazon.science/64/94/639ae0c44890837b0f1fbf11ef77/using-phoneme-representations-to-build-predictive-models-robust-to-asr-errors.pdf` seems correct URL. Let me know how to fix this.

**Comment by jflasher** [comment]
@anjiefang  It's likely just our schema regex having issues, I'll take a look.

**Comment by jflasher** [comment]
Ok, schema issue resolved. Longer TLDs were causing an issue.

File: datasets/asr-error-robustness.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Name: ASR Error Robustness
```
Comment (zflamig) [inline datasets/asr-error-robustness.yaml:1]: Can we expand ASR to Automated Speech Recognition (ASR) here so people can understand better what is contained in this dataset

Comment (anjiefang) [reply datasets/asr-error-robustness.yaml]: Thanks. Done.

**Comment by zflamig** [comment]
Thanks!

**Pull request merged**